src/css/colors.css

```diff
@@ -0,0 +1,9 @@
+@value primaryColor: #283756;
```

Comment:
We already have colors setup in `tailwind`, see #2. 
Can you use colors from there or add to if you need more colors.

Or are you trying to make the component independent which makes sense?

Reply:
Oh cool, I'd use that then. Didn't know. I'd update the other PRs too